.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
+language: go
+go:
+  - "1.11.x"
+  - "1.12.x"
+services:
+  - docker
+
+matrix:
+  allow_failures:
+  - go: tip
+
+env:
+  - GO111MODULE=on
+  - GO111MODULE=off
+
+addons:
+  # ssh_known_hosts: github.com
```

Comment:
Why is this commented out at the moment?  What purpose did/will it serve?

Reply:
[Cargo-culted](https://github.com/olivere/elastic/blob/release-branch.v7/.travis.yml#L14). Probably not needed.

- [ ] Remove `ssh_known_hosts` from .travis.yml